otx/cli/builder/builder.py

```diff
@@ -200,14 +200,23 @@ def build_task_config(self, task_type, model_type=None, workspace_path=None, otx
         model_config = MPAConfig.fromfile(os.path.join(template_dir, "model.py"))
         model_config.dump(os.path.join(workspace_path, "model.py"))
 
-        # Copy Data config
+        # Copy Data pipeline config
         if os.path.exists(os.path.join(template_dir, "data_pipeline.py")):
             data_pipeline_config = MPAConfig.fromfile(os.path.join(template_dir, "data_pipeline.py"))
             data_pipeline_config.dump(os.path.join(workspace_path, "data_pipeline.py"))
+        elif os.path.exists(os.path.join(template_dir, template_config.base_data_pipeline_path)):
+            data_pipeline_config = MPAConfig.fromfile(
+                os.path.join(template_dir, template_config.base_data_pipeline_path)
+            )
+            data_pipeline_config.dump(os.path.join(workspace_path, "data_pipeline.py"))
```

Comment:
I recommend to correct here so as to the above.

Reply:
I will modify it in such a way that it does not directly search for `data_pipeline.py` in template, but only searches for `base_data_pipeline_path`. Thanks for the nice comment!